test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -45,11 +47,11 @@ TEST(help_add_test, add_option)
     EXPECT_THROW(parser1.parse(), parser_interruption);
     stdout = testing::internal::GetCapturedStdout();
     expected = std::string("test_parser"
-                                       "==========="
-                                       "VERSION"
-                                       "Last update:"
-                                       "test_parser version:"
-                                       "SeqAn version: 3.0.0");
+                           "==========="
+                           "VERSION"
+                           "Last update:"
+                           "test_parser version:"
+                           "SeqAn version: 3.0.0");
```

Comment:
include the actual seqan3 version from seqan3/version.hpp instead of hard-coding